lib/UserManagement.php

```diff
@@ -68,19 +68,32 @@ public function getUser($userId)
      * @param string|null $firstName The first name of the user.
      * @param string|null $lastName The last name of the user.
      * @param boolean|null $emailVerified A boolean declaring if the user's email has been verified.
+     * @param string|null $password The password to set for the user.
+     * @param string|null $passwordHash The hashed password to set for the user.
+     * @param string|null $passwordHashType The algorithm originally used to hash the password. Valid values are bcrypt.
```

Comment:
worth adding an enum for `passwordHashType`?

Reply:
🤔 when/if there's more than one one day.